products/jbrowse-web/src/tests/BookmarkWidget.test.tsx

```diff
@@ -62,8 +62,8 @@ test('using the hotkey to bookmark the current region', async () => {
 
   // @ts-expect-error
   const { bookmarks } = session.widgets.get('GridBookmark')
-  expect(bookmarks[0].start).toBe(100)
-  expect(bookmarks[0].end).toBe(140)
+  expect(bookmarks[0].start).toBe(105)
```

Comment:
is this expected to be changed? could be related to https://github.com/GMOD/jbrowse-components/issues/3996 ?

Reply:
we should likely do a full snapshot of the bookmarks object for test